README.md

```diff
@@ -13,24 +13,24 @@ remote Opam repository:
 
 ## Layout of This Repository
 
-Packages are organised into namespaces. They are equivalent from the
-perspective of the [Opam] user.
+Packages are organised into namespaces:
 
 * `upstream`: packages that we don't control with fixed versions.
+* `upstream-extra`: packages that we don't control with fixed versions.
 * `xs`: packages that we control, with fixed versions.
 * `xs-extra`: packages that we control, following their respective
   master branch.
 
+The namespaces are treated differently as part of the build process of
+XenServer components but this should not concern an Opam user - packages
+in all namespaces together form the repository and there is no
+difference between them.
+
     ```
-    ./packages
```

Comment:
For some reason this block is not rendered correctly, the backticks are visible in the output: https://github.com/lindig/xs-opam/